schematic/models/validate_attribute.py

```diff
@@ -1533,13 +1607,13 @@ def _gather_value_warnings_errors(
 
     def _check_if_target_manifest_is_empty(
```

Comment:
I am not seeing unit test for `_check_if_target_manifest_is_empty`? If this can't be added in this PR, can we open a ticket so that we don't lose track?

Reply:
Same as above the whole file needs(as much of Schematic) needs unit tests.